index.js

```diff
@@ -23,68 +26,72 @@ function getRandomNumber(min, max) {
 }
 
 function checkGuess() {
-  // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
 
   hideAllMessages();
 
-  if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+  if (guess < guessInput.min) {
+    wayTooLow.style.display = "block";
+  } else if (guess > guessInput.max) {
+    wayTooHigh.style.display = "block";
```

Comment:
Nice job adding some extra conditions here, but they introduced one more very sneaky bug, which is that guessing above 100 or below 0 still consumes a guess!